pkg/trace/obfuscate/sql.go

```diff
@@ -206,20 +217,23 @@ type tableFinderFilter struct {
 // Filter implements tokenFilter.
 func (f *tableFinderFilter) Filter(token, lastToken TokenKind, buffer []byte) (TokenKind, []byte, error) {
 	switch lastToken {
-	case From:
+	case From, Join:
 		// SELECT ... FROM [tableName]
 		// DELETE FROM [tableName]
+		// ... JOIN [tableName]
 		if r, _ := utf8.DecodeRune(buffer); !unicode.IsLetter(r) {
 			// first character in buffer is not a letter; we might have a nested
 			// query like SELECT * FROM (SELECT ...)
 			break
 		}
 		fallthrough
-	case Update, Into, Join:
+	case Update, Into:
 		// UPDATE [tableName]
 		// INSERT INTO [tableName]
-		// ... JOIN [tableName]
-		f.storeName(string(buffer))
+		if f.storeTableNames {
+			f.storeName(string(buffer))
+		}
+		return Table, buffer, nil
```

Comment:
```suggestion
		return TableName, buffer, nil
```
I think `TableName` is nicer and clearer. But just a nit, feel free to ignore if you think otherwise.

Reply:
I think `Table` is more consistent with the other tokens and is the same name used as the tokenizer this is based on https://github.com/blastrain/vitess-sqlparser/blob/develop/sqlparser/sql.go#L162

Reply:
Actually it's not. `Table` there refers to the SQL keyword `TABLE`. That's exactly the reason I made the comment. Check this line for example: https://github.com/blastrain/vitess-sqlparser/blob/develop/sqlparser/sql.go#L48. They also use `TableName`

Reply:
Ah, my mistake! I agree with your rationale then. Updated and also changed the naming from normalize to match current conventions.